mitodl/micromasters-zendesk-theme
# feat: update logo

## Issues
Closes https://github.com/mitodl/hq/issues/3798

## Description


Updates logos in the header and footer

## Testing instructions

- Verify that the header and footer logos match https://micromasters.mit.edu
- You can do this locally by following the Readme to setup and preview the theme
- OR you can verify on my trial instance https://arbisoft4369.zendesk.com/hc/en-us 

## Screenshots


![Opera Snapshot_2024-03-22_162141_arbisoft4369 zendesk com](https://github.com/mitodl/micromasters-zendesk-theme-updated/assets/52656433/89aa3ea9-d9ad-4063-a55f-fd9c6d3a7749)
